src/View/CsvView.php

```diff
@@ -373,6 +383,12 @@ protected function _generateRow($row = null)
             $csv = str_replace("\n", $eol, $csv);
         }
 
+        $dataEncoding = $this->viewVars['_dataEncoding'];
+        $csvEncoding = $this->viewVars['_csvEncoding'];
+        if ($dataEncoding != $csvEncoding) {
```

Comment:
Can you give me some help ho to update this? Shall I push my change again and ask for a PR once more?